.github/workflows/test-generated-project.yml

```diff
@@ -0,0 +1,56 @@
+name: Test Generated Project
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+env:
+  TERRAFORM_VERSION: "1.5.5"
+  TFSEC_VERSION: "v1.28.1"
+
+jobs:
+  test:
+    name: Run Tests Generated Project
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x]
```

Comment:
I think we don't need to run the test with 2 different node versions:
- what we test is mainly about `terraform fmt -recursive -check`, and it's not about the compatibility with a specific node version
- Node 16 will [stop being supported in April 2024](terraform fmt -recursive -check
), so let's only use 18+ versions from now on

Reply:
I'd prefer to save some resources and test only on version 18. Since version 16 reached EOL already and it will be deprecated soon.

Reply:
Removed it out in [add9bd2](https://github.com/nimblehq/infrastructure-templates/pull/276/commits/add9bd2b01d1cad45d64496943de5dc827cc3d14)